public/stylesheets/signup.css

```diff
@@ -0,0 +1,19 @@
+.signup-form {
+  padding-top: 2rem;
+  padding-bottom: 8rem;
+}
+
+.signup-form-field {
+  padding-bottom: 1rem;
+}
+
+.signup-form img {
+  width: 125px;
+  margin: 0 auto;
+  display: block;
+}
+
+.signup-form .btn.btn-primary {
+  width: 100%;
+  margin-top: 1rem;
+}
```

Comment:
```suggestion
}

.signup-form input {
  border-color: var(--button-primary);
}

```

Reply:
@lyrated @simbuzan overall looks greatt 
this border color is just to match the design on figma :D